### ISSUE TITLE
Can not stop after one "stop start" cycle 

### ISSUE THREAD

**Issue by smclelland** [mojombo/god#91]
Simple watch on node.js.  I have node writing out a pid file on start and pid_file param is pointing to it.   Starting god up using -D, everything fires up perfectly.   I threw in some debug output in a proc on the .start/.stop to see what's going on. 

"god -c ..."
-> myproc up

"god stop myproc"
-> debug stop:  killing (good so far)

"god status"
myproc: unmonitored (good so far)

"god start myproc"
-> debug start:  starting (good so far)

"god stop myproc"
???????  (no so good)

"god status"
myproc: up (huh? up?)

Looking at the god console.  There is no "[ok] process is running for 'myproc'", yet status says its up?

Running on ubuntu 11 and ruby 1.9.2.

Basically I have to terminate god and manually kill the processes and start everything back up then I get one round of stopping/starting.
